### DISCUSSION TITLE
朋友爱上邦布怎么办？

### DISCUSSION THREAD

**Discussion by 1122246** [share121/inter-knot#611] [其他]
我有一个朋友，现在已经沉迷邦布无法自拔了……整条街的邦布都被他骚扰过！！！
昨天刚被治安布当成可疑人员送进治安局，还要我去接他，八辈子的脸都丢完了！！！怎样才能让他悔改？？？

**Comment by Tragedysaver1024** [comment]
这位先生你好，我有位朋友在在通辽圣伊丽莎白精神病院当主治医生，您不妨送他去那里看看？

**Comment by WRYYY-S** [comment]
你好我是治安官，你的这个朋友已经没救了，我们局的治安布差点被他恶心到不能正常出门了

**Comment by Tragedysaver1024** [comment]
> 这位先生你好，我有位朋友在在通辽圣伊丽莎白精神病院当主治医生，您不妨送他去那里看看？

据我所知去过那里的人精神都很正常，除了有那么一点点小小的后遗症。
